README.md

```diff
@@ -227,7 +227,13 @@ Future work and Limitations
 Change Log (From version 2.2.0 and onwards)
 ==============
 
-### __2.6.0-SNAPSHOT__
+### __3.0.0__
+
+* Upgraded to Hive 1.2.1 (Note: new major release with backwards incompatibility issues). As of Hive 1.2 there are a number of new reserved keywords, see [DDL manual](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Keywords,Non-reservedKeywordsandReservedKeywords) for more information. 
+If you happen to have one of these as an identifier, you could either backtick quote them (e.g. \`date\`, \`timestamp\` or \`update\`) or set hive.support.sql11.reserved.keywords=false.                                            
+* Removed the custom HiveConf hive.vs. Use hadoop.tmp.dir instead.   
```

Comment:
Hmm, that's a shame as that would prevent people using older versions of Hive from upgrading. I can imagine that this kind of backward/forward compatibility across different versions of Hive must be tricky to do. Not sure what the best path forward is?